### PR TITLE
fix(trigger): add label for workflowRun created by trigger

### PR DESCRIPTION
**Pull request by tskdsb**
**What this PR does / why we need it**:

- 给trigger创建的wfr加上label，使之能被list请求到
- 相关issue: https://github.com/caicloud/clever-quality/issues/375

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/assign @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/workflow/controller/handlers/workflowtrigger/cron_job.go [hunk 1]
```diff
@@ -91,6 +92,12 @@ func ToWorkflowTrigger(obj interface{}) (*v1alpha1.WorkflowTrigger, error) {
 func (c *CronTrigger) Run() {
 
 	c.WorkflowRun.Name = c.WorkflowTriggerName + "-" + strings.Replace(uuid.NewV1().String(), "-", "", -1)
+
+	if c.WorkflowRun.Labels == nil {
+		c.WorkflowRun.Labels = make(map[string]string)
+	}
+	c.WorkflowRun.Labels[CleverWorkflowName] = c.WorkflowRun.Spec.WorkflowRef.Name
```
Comment (cd1989) [inline pkg/workflow/controller/handlers/workflowtrigger/cron_job.go]: Please use annotations to add workflow name to WorkflowRun.

File: pkg/workflow/controller/handlers/workflowtrigger/cron_job.go [hunk 2]
```diff
@@ -23,6 +23,7 @@ const (
 	// -480 for Asia/Shanghai(+8)
 	ParamTimeZoneOffset = "timeZoneOffset"
 	ParamSchedule       = "schedule"
+	CleverWorkflowName  = "clever.caicloud.io/workflow-name"
```
Comment (cd1989) [inline pkg/workflow/controller/handlers/workflowtrigger/cron_job.go]: Please define const in `pkg/workflow/common/const.go` follow `WorkflowRunAnnotationName`, like 

```
WorkflowRunAnnotationName = "cyclone.io/workflowrun"
WorkflowAnnotationName = "cyclone.io/workflow"
```

Don't use `clever`, `caicloud` terms as we are creating a open source project.

Comment (tskdsb) [reply pkg/workflow/controller/handlers/workflowtrigger/cron_job.go]: OK, I will move it and use `cyclone.io/workflow`

File: pkg/workflow/controller/handlers/workflowtrigger/cron_job.go (hunk 1 above)
Comment (tskdsb) [reply pkg/workflow/controller/handlers/workflowtrigger/cron_job.go]: no, my leader asked me to use `label` instead of annotations

@codeflitting

**Pull request merged**